sensor_msgs/Image image
float32[] feature
geometry_msgs/PointStamped camera
